docs/en/concepts-and-designs/OAP-Cluster-Management.md

```diff
@@ -0,0 +1,39 @@
+# Observability Analysis Platform Cluster Management
+OAP(Observability Analysis Platform) server is a distributed system, services need to find each 
+other. i.e. a web service needs to find query service, level 1 aggregate service need to find 
+level 2 aggregate service. Cluster management is just a client-side implementation. It must work 
+together with a distributed coordination server, (i.e. Zookeeper, Consul, Kubernetes.) unless 
+running in standalone mode.
+
+## Architecture overview
+<img src="https://skywalkingtest.github.io/page-resources/6.0.0-alpha/cluster_management.png"/>
+
+### Cluster Management Plugins
+By default, OAP server provides two implementations for cluster management, which are standalone 
+and zookeeper. When the applications being monitored are small, you can choose the standalone mode.
+If they are big, you must choose the cluster mode by zookeeper plugin, or you can implement another 
+service discovery plugin for your own scene.
+
+### Cluster Management Interface
+There are two interfaces defined beforehand in the OAP server core, which are Module register and 
+module query, all the cluster management plugins must implement those two interfaces.
+
+* Module Register: When any modules which need to provide services for each other, those modules 
```

Comment:
`When any module needs to provide services for others, the module must do register through this interface.` 
The first half of adjustment is about the language only, the second half is that Cluster Management Interface can't guarantee the register will be into service discovery service. Like `standalone` is no service existed outside, `Kubernetes ` is doing service instance management by itself, no real register happens.